pkg/kubelet/nodeconfig/controller.go

```diff
@@ -0,0 +1,270 @@
+/*
```

Comment:
Please make sure that this (and any other new files) are uniquely named and not generic like controller.go, as it makes interpreting log messages in log files quite difficult. In other words, please name this nodeconfig_controller.go.